docs/src/lang/booleans.md

```diff
@@ -350,7 +350,7 @@ FALSE <=> 1      \* runtime error in TLC, type error in Apalache
 
 **Example in Python:**
 
-Assuming that both expressions are Boolean, `F <=> G` is equivalent to `F = G`.
+Assuming that both expressions are Boolean, `F <=> G` is equivalent to `F == G`.
```

Comment:
I think this equivalence refers to TLA+, where equality is written with `=`.

Reply:
It is under "Example in Python" though?

Reply:
Oh... since it is under _Example in Python_ I read this as "Assuming that both expressions are Boolean, `F <=> G` [in TLA] is equivalent to `F == G` [in Python]. Should we keep it as-is?

Reply:
Good question. Perhaps, add "in TLA+" in both cases and say that as a result we can use the Python equality '=='?

Reply:
It is indeed confusing.

Reply:
Fixed in [fbf3e21](https://github.com/informalsystems/apalache/pull/1328/commits/fbf3e216592cae2952c13e0bb10e833461acbdab) below